lib/user.rb

```diff
@@ -30,12 +30,12 @@ def password_from_word_list
     WORD_LIST.sample(3).map(&:capitalize).join
   end
 
-  def create_user(email)
+  def create_user(contact)
```

Comment:
This is referred to as a model in the tests, surprising to me to see a create_user method on this object.